composer.json

```diff
@@ -36,7 +36,7 @@
         "mockery/mockery": "^1.0",
         "mongodb/laravel-mongodb": "^5.0",
         "orchestra/testbench": "^8.0|^9.0|^10.0",
-        "orchestra/testbench-core": "^8.0|^9.0|^10.0",
+        "orchestra/testbench-core": "^8.30|^9.7|^10.0",
```

Comment:
These are the first versions that the `package:sync-skeleton` command exists, which we depend on. This shouldn't impact package compatibility as it's a dev dependency.